### ISSUE TITLE
Suggestion: option to disable some of the console debugging

### ISSUE THREAD

**Issue by rdster** [TangoTek/TekTopia-Community#979]
The console gets pretty flooded with TT output.

I  think having some way to select what is shown in the console vs placed in the log would be a great help.

The basic msgs of village construct and destruct are great for debugging, they aren't really needed in a live environment console. (I'd say the same with entity attachment to villages...and skill up, and basic action msgs.)

However, error messages, or villager damage msgs would be nice to have still...in case someone is going around killing off other villagers.

**Comment by smmmadden** [comment]
I agree @rdster, the debug lines should only be for turning on debug.  Having 15mb+ log files each day for only two people (wife and I) on my server currently is a problem going live once we're done.
![image](https://user-images.githubusercontent.com/10309346/64078819-a28bf300-ccad-11e9-9138-2fa11d6d5184.png)
